source/CardGames.py

```diff
@@ -243,21 +243,88 @@ def printHands(self, players: dict):
         player.showHand(True)
         print() 
 class GoFish:
-  def __init__(self, players: "list[Player]"):
+  def __init__(self):
     self.playerinfo = []
-    self.players = players
+    self.players = []
     self.value_wanted = int
     self.suit_wanted = ""
     self.player_asked = ""
     self.player_turn = ""
     self.info_player_turn = int
     self.info_player_asked = int
 
+  def multiplayer_info(self):
```

Comment:
Code looks good! Appreciate the comments they helped me understand your approach more clearly.

Reply:
Thanks Andre, glad they helped! I'll continue to add comments so that it's easier to understand.